kedify-agent/values.yaml

```diff
@@ -240,3 +240,9 @@ keda-add-ons-http:
   interceptor:
     scaledObject:
       waitForCrd: true
+otelAddOn:
```

Comment:
just nit:

`otelAddOn` vs `keda-add-ons-http` is not very consistent imho. Do we know if there's anyone using the uberchart already? if not, we can probably change the not so nice `keda-add-ons-http` to something like `httpAddOn`

Feel free to disregard, this is just my OCD

Reply:
yeah, we have users deploying kedify-agen chart with dependencies, iirc some enable addon through camel case option and some through snake case, not sure which one was first but the second option was on demand from a user.